Dashboard/Views/MainWindowView.axaml

```diff
@@ -0,0 +1,60 @@
+﻿<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:views="clr-namespace:Dashboard.Views"
+        xmlns:vm="clr-namespace:Dashboard.ViewModels"
+        xmlns:models="clr-namespace:Dashboard.Models"
+        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+        x:Class="Dashboard.Views.MainWindowView"
+        x:DataType="vm:MainWindowViewModel"
+        Title="MainWindow"
```

Comment:
This title will need to be changed
also plz add back the window logo (maybe change to nuvotion logo?):
```suggestion
        Title="MainWindow"
        Icon="/Assets/avalonia-logo.ico"
```

Reply:
Will do.

Reply:
19b1f940a5ff9e1743d7213780f9944f81b4fd27